### PR TITLE
Support PHP runtime 8.0.22 and 8.1.9

### PR DESCRIPTION
**Pull request by qianz2**
- [ ] The purpose of this PR is explained in this message or in an issue. If an issue please include a reference as \#\<issue\_number\>.
- [ ] Tests are included and/or updated for code changes.
- [ ] Proper license headers are included in each file.

### CODE REVIEW

File: doc/supportedPlatformVerions.md [hunk 1]
```diff
@@ -0,0 +1,949 @@
+# Supported platforms and versions
```
Comment (pauld-msft) [inline doc/supportedPlatformVerions.md:1]: I think we can remove this misspelled file in favor of the correctly named one thats modified in this PR

Comment (qianz2) [reply doc/supportedPlatformVerions.md]: Updated

File: build/__phpVersions.sh [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
+PHP_FPM_RUNTIME_BASE_TAG='20220827.1'
 COMPOSER_VERSION='2.0.8'
 COMPOSER_SETUP_SHA384='55ce33d7678c5a611085589f1f3ddf8b3c52d662cd01d4ba75c0ee0459970c2200a51f492d557530c71c15d8dba01eae'
 COMPOSER1_9_VERSION='1.9.3'
 COMPOSER1_10_VERSION='1.10.19'
 COMPOSER2_0_VERSION='2.0.8'
 COMPOSER2_2_VERSION='2.2.9'
 COMPOSER2_3_VERSION='2.3.4'
-PHP81_VERSION='8.1.6'
+PHP81_VERSION='8.1.9'
```
Comment (william-msft) [inline build/__phpVersions.sh:12]: @qianz2 should we update our integration tests for these new versions now?

Comment (cormacpayne) [reply build/__phpVersions.sh:12]: @william-msft it should be updated automatically with the tests referencing the `Php81Version` variable, correct?

Comment (qianz2) [reply build/__phpVersions.sh:12]: @william-msft I checked the integration tests, they're referring to the major minor version constants so I think it's updated.

Comment (william-msft) [reply build/__phpVersions.sh:12]: Ah yes, since it's a minor version bump.

**Review by william-msft — APPROVED** [review]

**Pull request merged**